src/fft_2n.h

```diff
@@ -207,6 +207,7 @@ void Radix2<T>::fft(vec::Vector<T>& output, vec::Vector<T>& input)
 }
 
 /** Perform decimation-in-frequency FFT or inverse FFT
+ * Note: input vector is in reversed-bit order
```

Comment:
Skip a line and use Doxygen syntax (`@note`, or even `@pre` if it's a pre-condition).